Comps/Top.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useState } from 'react';
+
+import {View, Text, Switch} from 'react-native';
+
+
+function Top() {
+	const [value, onChangeValue] = React.useState('White');
+	
+	
+  return (
+  
+	  <View style={{flex:1, backgroundColor:{value}, alignItems:'center', justifyContent:'center' }}>
+	  
+		<Text>{value} Mode</Text>
+		
+		 <Switch
+		 onValueChange = {
+			 value => onChangeValue('Black')
+		 }
+		  />
+	  </View>
+	  
+  );
+}
+export default Top;
```

Comment:
Is it possible to change the indentation from four to two spaces so that it is consistent with the rest of the code. Otherwise the code looks good to me.